sonar-update-center-mojo/src/test/java/org/sonar/updatecenter/mojo/GenerateHtmlMojoTest.java

```diff
@@ -46,6 +46,10 @@ public void generate_prod_html_without_private() throws Exception {
 
     File compatibilityMatrix = new File(outputDir, "html/compatibility-matrix.html");
     assertThat(compatibilityMatrix).exists().isFile();
+    compatibilityMatrix = new File(outputDir, "html/compatibility-matrix-sqs.html");
+    assertThat(compatibilityMatrix).exists().isFile();
+    compatibilityMatrix = new File(outputDir, "html/compatibility-matrix-sqcb.html");
+    assertThat(compatibilityMatrix).exists().isFile();
```

Comment:
Here the compatibilityMatrix is being over written twice.
The next assertion to validate that the HTML contains "Artifact Size" is only performed on the SQCB HTML.

Maybe both assertions can be extracted in a method that getting the file path as input asserts both the existence of the file and the content of it (At least the content of Artifact Size as now).